receiver/podmanreceiver/podman_client.go

```diff
@@ -57,10 +60,35 @@ type containerStatsReport struct {
 	Stats []containerStats
 }
 
+type Type = string
```

Comment:
It is used in the logs translation for creating the logs body.

Reply:
I tried to fine it but couldn't. Could you please link to it? 

1. If we MUST use this instead of `string`, can we please use a better name? 
2. Does this correlate to the event.Type field? If so, why not similar type for `Action` and probably should be named `eventType` or something that is self explanatory.

Reply:
In the initial approach (before the first commit), we were using it for `event.Type`, but as the type is only string, we directly assigned the string type to `event.Type` so we no longer needed it. It is removed in the latest commit.